.gitignore

```diff
@@ -3,3 +3,4 @@ yarn-error.log
 
 # macOS
 .DS_Store
+.yarnrc
```

Comment:
This file will be removed on the next `git clean -xdf`. Feel free to check it in instead